### PR TITLE
Update Develop with 3.6-rc changes

### PR DESCRIPTION
**Pull request by MrPetovan**

### CODE REVIEW

File: boot.php [hunk 1]
```diff
@@ -38,8 +38,8 @@
 require_once 'include/text.php';
 
 define('FRIENDICA_PLATFORM',     'Friendica');
-define('FRIENDICA_CODENAME',     'Asparagus');
-define('FRIENDICA_VERSION',      '3.6-dev');
+define('FRIENDICA_CODENAME',     'The Tazmans Flax-lily');
+define('FRIENDICA_VERSION',      '3.6');
```
Comment (tobiasd) [inline boot.php]: The version should now be `2018-5-dev` I think.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-3.6-dev
+3.6
```
Comment (tobiasd) [inline VERSION]: The version should now be `2018-5-dev`

**Comment by MrPetovan** [comment]
Ha, I looked for "asparagus" but I didn't think about the actual version number, it is updated now.

**Comment by annando** [comment]
BTW: Had you got a look at the ACL problem? that Hoergen described?

**Comment by MrPetovan** [comment]
No, this morning I only solved the conflicts, this can come after.

**Comment by annando** [comment]
Okay. So I'm currently stuck on master :-)

**Comment by MrPetovan** [comment]
I'm not even sure what the problem is, because no issue has been created, so the problem now is in a forum thread buried deep under all the release candidate posts.

**Comment by annando** [comment]
The problem is that he creates a non public post and includes a single receiver, but additionally the main contact group is added.

**Comment by MrPetovan** [comment]
I can't reproduce this on the current develop, no reason why it would happen once rc is merged.

**Comment by tobiasd** [comment]
> I'm not even sure what the problem is, because no issue has been
> created, so the problem now is in a forum thread buried deep under
> all the release candidate posts.

See #4616 for the issue opened by @hoergen

**Comment by MrPetovan** [comment]
My bad, I was the one asking him to open an issue 🤦‍♂️

**Pull request merged**